worldedit-core/src/main/java/com/sk89q/worldedit/internal/registry/InputParser.java

```diff
@@ -33,10 +36,18 @@
 
     protected final WorldEdit worldEdit;
 
-    protected InputParser(WorldEdit worldEdit) {
+    public InputParser(WorldEdit worldEdit) {
         this.worldEdit = worldEdit;
     }
 
     public abstract E parseFromInput(String input, ParserContext context) throws InputParseException;
 
+    /**
+     * Gets a list of suggestions of input to this parser.
+     *
+     * @return a list of suggestions
+     */
+    public List<String> getSuggestions() {
+        return Lists.newArrayList();
```

Comment:
I would prefer a use of `Collections.emptyList()` here, there's no need for this array list.